curriculum/challenges/english/07-scientific-computing-with-python/learn-tree-traversal-by-building-a-binary-search-tree/65ca06475e9b8aff8ea5a5f1.md

```diff
@@ -1,13 +1,13 @@
 ---
-id: 655a4761e1a40065fc4d3712
-title: Step 31
+id: 65ca06475e9b8aff8ea5a5f1
+title: Step 34
 challengeType: 20
-dashedName: step-31
+dashedName: step-34
 ---
 
 # --description--
 
-After your existing `if`, write another one that checks if the target key is less than the current node key.
+After your existing `if`, write another one that checks if the target key is less than the current node key. Remember to use `pass`.
```

Comment:
At some point we should be able to avoid this reminder, and maybe put it as an hint if necessary

Reply:
Such instructions are now in the hint only. This is an advanced project and we can skip telling the campers to do this in every instruction.